src/main/java/casper/tasks/ToDo.java

```diff
@@ -0,0 +1,49 @@
+package casper.tasks;
+
+/**
+ * Represents a "ToDo" task, which is a type of task that only requires a description.
+ * Inherits the basic task functionalities from the {@link Task} class.
+ */
+public class ToDo extends Task{
```

Comment:
Should a space be added between { and Task?

Reply:
I do think so. Thanks for pointing out!